Syntax.in

```diff
@@ -0,0 +1,41 @@
+<type> ::= "number" | "string" | <array_declaration>
+
+<array_declaration> ::= "number" "[" <integer> "]"
+
+<readstmt> ::= "read(" <identifier> ")"
+
+<declaration> ::= <type> " " <identifier>
+
+<decllist> ::= <declaration> | <declaration> ", " <decllist>
+
+<printstmt> ::= "print(" (<identifier> | <constant>) ")"
+
+<relation> ::= "<" | "<=" | "==" | "!=" | ">=" | ">"
+
+<expression> ::= (<expression> "+" | "-" <term>) | <term>
```

Comment:
aici si mai jos oare n-ar trebui puse "+" | "-" intr-o paranteza?
de fapt, in BNF nu sunt paranteze. cred ca trebuie sa scrii o data cu +,  un sau, si acelasi lucru cu  -.